### ISSUE TITLE
FE Searching: Only germany

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#196]
## Description
See Epic: #193 
Searching an address displays to many options wich are not relevant

## Acceptance criteria
* Only search for addresses / coordintes within germany

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
